### ISSUE TITLE
RQ3: How to find Bellwether? 

### ISSUE THREAD

**Issue by vivekaxl** [vivekaxl/Bellwether-Config#17]
### Results:

sqlite : 3.682 +- 4.647
spear : 0.099 +- 0.0
sac : 0.639 +- **27.884**
storm-obj2 : 0.0 +- 0.0
x264 : 1.511 +- 8.182

### Doesn't work:
storm-obj1 : 2.268 +- **48.616**

### Notes:
- The rank predicted by the bellwether found by the bellwether method in x264 is within the top 1.51% of the dataset.  
- This can also be interpreted as the distance to heaven, where the distance is normalized to the range of the dependent values.

**Comment by pooyanjamshidi** [comment]
One thing that we need to consider is the variance in the measurements in datasets, basically the distance between the best and worst configurations. The distance of worst configuration to heaven from one dataset vary to another depending on the variance, right?

**Comment by vivekaxl** [comment]
Correct. I have normalized the dependent values of all the datasets.
